code/learning_material/MessageList.js

```diff
@@ -0,0 +1,69 @@
+// This is all the material from the Monday lecture
+
+import React, { useEffect, useState } from 'react';
+import moment from "moment";
+
+export const MessageList = () => {
```

Comment:
Are you using this component at all? I see another one, `<Card />` component below. One of them should be deleted